src/libstore/build/local-derivation-goal.hh

```diff
@@ -86,8 +86,8 @@ struct LocalDerivationGoal : public DerivationGoal
             : source(source), optional(optional)
         { }
     };
-    typedef map<Path, ChrootPath> DirsInChroot; // maps target path to source path
-    DirsInChroot dirsInChroot;
+    typedef map<Path, ChrootPath> FilesAndDirsInChroot; // maps target path to source path
+    FilesAndDirsInChroot filesAndDirsInChroot;
```

Comment:
Maybe just "PathsInChroot" to be less verbose?

Reply:
Done